package.json

```diff
@@ -1162,10 +1169,12 @@
   "dependencies": {
     "@brightside/core": "^2.36.0",
     "@types/fs-extra": "^7.0.0",
+    "@zowe/imperative": "^4.4.4",
```

Comment:
Do we need to import imperative?
I thought we got it already from `@brightside/core`

Reply:
No I thought we got it from imperative but that was just a dev dependency. I trust this is till okay. It only gets pulled in if there is a need to go secure

Reply:
I don't believe we need it now and will take it out.